content/docs/guides/responsive_amp/placeholders.md

```diff
@@ -62,6 +62,21 @@ that the browser doesn’t support a particular feature:
 The `fallback` attribute can be set on any HTML element, not just AMP elements.
 If specified, the `fallback` element must be a direct child of the AMP element.
 
+## Interaction of placeholders and fallbacks
+
+For AMP components that rely on dynamic content (e.g., `amp-twitter`, `amp-list`), the interaction of fallbacks and placeholders operates as follows:
+ 
+<ol>
+  <li>Display the placeholder while the content is loading.</li>
+  <li>If the content loads successfully, hide the placeholder and display the content.</li>
+  <li>If the content fails to load:
+    <ol>
+      <li>If there's a fallback element, display the fallback.</li>
+      <li>Otherwise, if there's a placeholder, display the placeholder.</li>
```

Comment:
Only, if placeholder is present, right?  If there's no placeholder, no fallback, and the content loads, display nothing. So, "Otherwise, continue displaying the placeholder (if present)."